docs/src/pages/components/slider/RangeSlider.js

```diff
@@ -14,13 +14,13 @@ export default function RangeSlider() {
   };
 
   return (
-    <Box sx={{ width: 300 }}>
+    <Box sx={{ width: 300, mt: 4 }}>
       <Slider
         getAriaLabel={() => 'Temperature range'}
         value={value}
         onChange={handleChange}
         valueLabelDisplay="auto"
-        getAriaValueText={valuetext}
```

Comment:
Why are you deleting this?

Reply:
`getAriaValueText` and `valueLabelFormat` are not the same though. You can keep this while adding `valueLabelFormat`.

Reply:
> `getAriaValueText` and `valueLabelFormat` are not the same though. You can keep `getAriaValueText` while adding `valueLabelFormat`.

yes, perhaps you are right